apps/src/blockTooltips/tooltipUtils.js

```diff
@@ -90,8 +90,15 @@ exports.findFunctionAndParamNumber = function (editor, position) {
           var isBeginningOfFunctionCall =
             seenCloserStack.length === 0 && currentOpener === '(';
           if (isBeginningOfFunctionCall) {
+            // if we have text "foo.bar(", funcName is foo.bar rather than just bar
+            var funcName = iterator.stepBackward().value;
+            var previousToken = iterator.stepBackward();
+            if (previousToken && previousToken.value === '.') {
+              funcName = iterator.stepBackward().value + '.' + funcName;
```

Comment:
Could be worth manually bashing a bit in the editor putting a bunch of dots in different situations (near newlines, left/right parens, other dots, etc.) just to see if anything goes wrong. Looks like it should be fine though!